ecocode-rules-specifications/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>io.ecocode</groupId>
 		<artifactId>ecocode-parent</artifactId>
-		<version>1.4.8-SNAPSHOT</version>
+		<version>1.4.9-SNAPSHOT</version>
```

Comment:
as mentionned on my review on ecoCode-java plugin, no need to update this version ... this will be done during release process in a standard use case
but that you can done is update to 1.5.0-SNAPHOST because of deletion of deprecated rules